packages/code-du-travail-frontend/src/search/search.service.js

```diff
@@ -34,7 +34,10 @@ const suggestMin = query =>
 // memoize search results
 const fetchSearchResultsMemoized = memoizee(fetchSearchResults, {
   promise: true,
-  primitive: true,
+  // make sure the params get serialized correctly so memoize works well
+  normalizer: function(args) {
+    return JSON.stringify(args[0]);
+  },
```

Comment:
I wonder if we should rather pass 2 arguments (query / excludedSource) to keep primitive mode and
 avoid the normalizer

Reply:
True, I didn't want to bother with the params order but in the end I end up struggling with the memoize until I realised the two parameters would simplify this. So in the end it's about: is it better to have an configuration object and a normalized or two params ? I think I'm going to revert that one